umbraco-cms/reference/configuration/loggingsettings.md

```diff
@@ -1,25 +1,32 @@
 ---
-versionFrom: 9.0.0
-versionTo: 10.0.0
-meta.Title: "Umbraco Logging Settings"
-meta.Description: "Information on the logging settings section"
+description: "Information on the logging settings section."
 ---
 
 # Logging settings
 
 The majority of logging related configuration has been moved to the Serilog configuration see [Serilog settings](serilog.md) for more information.
 
-This settings section allows you to configure the max log age for the internal audit log scrubbing. The default maximum age for the internal audit log is 24 hours, however if you want to change this duration, you can with the `"MaxLogAge"` key, like so:
-
+The following configuration is available in the Logging settings:
 
 ```json
 "Umbraco": {
   "CMS": {
     "Logging": {
-      "MaxLogAge": "2.00:00:00"
+      "MaxLogAge": "2.00:00:00",
+      "Directory": "~/CustomLogFileLocation"
     }
   }
 }
 ```
 
-This will increase the maximum age of the entires in the audit log to 48 hours (2 days).
+## MaxLogAge
+
+This setting allows you to configure the maximum log age for the internal audit log scrubbing. The default maximum age for the internal audit log is 24 hours. Change the duration with the `"MaxLogAge"` key in the Logging settings.
+
+To increase the maximum age of the entires in the audit log to 48 hours (2 days), set the value to `2.00:00:00`.
+
+## Directory
+
+By default, all log files are saved to the `umbraco/Logs` directory. You can define a custom directory for your log files by using the `Directory` key in the Logging settings.
+
+The following value will add all log files to a `LogFiles` directory in the root of the file structure: `~/LogFiles`.
```

Comment:
The following value - Is there something missing here?

Reply:
The following value is at the end of the sentence.
If that's not clear, I'll change the structure of the sentence 💪

Reply:
ah ok.. If `~/LogFiles` is on a new line (unless it's on a new line and not showing on Github 🤔) it will make the following value read more clearly else you can rephrase it.